frontend/packages/dev-console/src/components/pipelines/modals/common/MultipleResourceKeySelector.tsx

```diff
@@ -93,7 +95,9 @@ const MultipleResourceKeySelector: React.FC<StateProps & MultipleResourceKeySele
         }}
         showBadge
       />
-      {field.value && <MultipleKeySelector name={resourceKeysField} keys={keys} />}
+      {field.value && (
```

Comment:
Interestingly found a bug while testing this - mind fixing it? It's should be a tiny one.

On [line 64](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/pipelines/modals/common/MultipleResourceKeySelector.tsx#L64) `keyMap` should have a default of `{}` to prevent an NPE on the `Object.keys` call.

I think we also would need to avoid rendering this block:
```suggestion
      {field.value && Object.keys(keys).length > 0 && (
```

Maybe you can think of another solution, this was my quick hack. Feel free to implement something else if you see a better fix for the NPE.